a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -381,23 +383,23 @@ <h4>Variables setup</h4>
                     
                 <!-- EPUB Accessibility 1.0 WCAG 2.0 (A/AA/AAA) -->
                  <li>
-                       <span><b>IF</b> the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="dcterms:<i>conformsTo</i>" and text() = "<i>http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a</i>"]</code> :</span>
+                     <span><b>IF</b> the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/link[@rel="dcterms:<i>conformsTo</i>" and @href="<i>http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a</i>"]</code> :</span>
```

Comment:
The specification used `link`, but did it didn't exclude people from using `meta`. I was thinking we should check for both patterns, but I think if you and Gregorio were requiring this pattern in your certification programs we're fine just testing for the one. Otherwise, these 1.0 tests need to combine both:

`/package/metadata/link[@rel="dcterms:conformsTo" and @href="http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"] or /package/metadata/meta[@property="dcterms:conformsTo" and text() = "http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"]`

Reply:
What do you think @gregoriopellegrino should I check for both?

Reply:
I just added in the recommended string-based version along with the URL based version for the 1.0 conformance metadata.

Reply:
Sorry, I just noticed I gave you the wrong xpath syntax. The "or" between the two element selectors should be a "|". I'll push a fix for you in a second.

Reply:
Thanks Matt